tests/program/Program.cpp

```diff
@@ -33,7 +33,7 @@ void DoSerializationTest(const std::string& filename) {
 
   REQUIRE( reconstituted == original );
 
-  os.close();
+  is.close();
```

Comment:
Can we delete this altogether? Shouldn’t ifsream’s destructor take care of this?

Reply:
can we?